### DISCUSSION TITLE
Welcome to Hacktoberfest-2024 Discussions!

### DISCUSSION THREAD

**Discussion by hari7261** [hari7261/Hacktoberfest-2024#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hari7261** [comment]
Contributions  should be in JavaScript.

**Comment by ashwin925** [comment]
ok, i understand. coming right up!!!

**Comment by himanshu9470** [comment]
Hello! Here with an updated answer:

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. If users create discussions of this nature, it may result in a temporary or indefinite block from the Community.

Thanks for understanding!

**Comment by hari7261** [comment]
> Hello! Here with an updated answer:
> 
> We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
> 
> Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. If users create discussions of this nature, it may result in a temporary or indefinite block from the Community.
> 
> Thanks for understanding!

Thanks for giving me valueable instruction about this issue
